### ISSUE TITLE
The "Currently on:" message still displays 25 blocks south of a location

### ISSUE THREAD

**Issue by Abelkrijgtalles** [Abelkrijgtalles/mojang-maps#3]
The "Currently on:" message still displays 25 blocks south of a location.
Images:
![2023-08-01_14 41 30](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/d0339e82-5f43-4b3a-ab4b-8825a7647cd1)
![2023-08-01_14 41 40](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/a91c1e86-0dd3-4177-bac4-6136864ed00d)
![2023-08-01_14 41 56](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/ae86d490-840e-4031-b3cc-eea99329bbc5)
![2023-08-01_14 42 13](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/40d8650d-401d-4c40-b669-9079978124cd)

**Comment by Abelkrijgtalles** [comment]
Btw my nodes.yml:
```
# This is auto-generated by Mojang Maps.
# Changing anything in this file will result in a broken Mojang Maps.

roads:
- ==: nl.abelkrijgtalles.MojangMaps.object.Road
  name: LekkaDorpieSquare
  locations:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
locations:
- ==: org.bukkit.Location
  world: world
  x: 64.0
  y: 103.0
  z: -432.0
  pitch: 0.0
  yaw: 0.0
- ==: org.bukkit.Location
  world: world
  x: 76.0
  y: 103.0
  z: -430.0
  pitch: 0.0
  yaw: 0.0
- ==: org.bukkit.Location
  world: world
  x: 76.0
  y: 105.0
  z: -445.0
  pitch: 0.0
  yaw: 0.0
- ==: org.bukkit.Location
  world: world
  x: 70.0
  y: 103.0
  z: -431.0
  pitch: 0.0
  yaw: 0.0
- ==: org.bukkit.Location
  world: world
  x: 76.0
  y: 103.0
  z: -435.0
  pitch: 0.0
  yaw: 0.0
- ==: org.bukkit.Location
  world: world
  x: 76.0
  y: 103.0
  z: -440.0
  pitch: 0.0
  yaw: 0.0

```

**Comment by Abelkrijgtalles** [comment]
it probably has something to do with isTheSameLocation()

**Comment by Abelkrijgtalles** [comment]
it seems like the gap is over 5/6

**Comment by Abelkrijgtalles** [comment]
it's the same location function that doesn't work

**Comment by Abelkrijgtalles** [comment]
Bug fixed and will be added in 1.4
